filebeat/fileset/fileset_test.go

```diff
@@ -232,51 +232,6 @@ func TestGetPipelineNginx(t *testing.T) {
 	assert.Contains(t, pipeline.contents, "processors")
 }
 
-func TestGetPipelineConvertTS(t *testing.T) {
```

Comment:
Seems like you need to cleanup the imports now (see CI).

Reply:
Oops